src/views/Entities/Zones.vue

```diff
@@ -339,11 +345,13 @@ export default {
     parseData(entity) {
       const { zoneInsight = {} } = entity
       let zoneCpVersion = '-'
+      let backend = ''
 
       if (zoneInsight.subscriptions && zoneInsight.subscriptions.length) {
         zoneInsight.subscriptions.forEach((item, index) => {
           if (item.version && item.version.kumaCp) {
             zoneCpVersion = item.version.kumaCp.version
+            backend = JSON.parse(item.config).store.type
```

Comment:
technically you can still have old Zone CP from the previous major so the config might be null.
I think it would safer to do `if (item.config) {`

Reply:
I will adjust it then